onedal/ensemble/forest.py

```diff
@@ -378,6 +393,15 @@ def _predict_proba(self, X, module, queue):
         return y
 
 
+# Version of register_hyperparameters that supports versioning
+register_hyperparameters_ver = daal_require_version_wrapper((2024, "P", 300))(
+    register_hyperparameters
+)
+
+
+@register_hyperparameters_ver(
+    {"predict": get_hyperparameters("decision_forest", "infer")}
+)
```

Comment:
```suggestion
@register_hyperparameters(
    {"predict": get_hyperparameters("decision_forest", "infer")}
)
```

Reply:
Could you comment on why the version check is not necessary? I forgot why I added it back in March. I probably followed examples in other classes. Are you sure it's safe to remove?